File: Server/5_Mission/Items/Dictionary/KCItemDesc.c
/// @brief Описание итема в справочнике
class KCItemDesc
{
    /// @brief Категория итема
    string Category;
    
    /// @brief Описание итема на языке котором запущен сервер
    string Description;
    
    
}

/// @brief Справочник итемов
typedef map<string, ref KCItemDesc>	KCItemDictionary;

File: .NET/KUBC.DAYZ.ITEMS/Dictionary/KCItemDesc.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace KUBC.DAYZ.ITEMS.Dictionary;

/// <summary>
/// Описание игрового предмета
/// </summary>
public class KCItemDesc
{
    /// <summary>
    /// Категория игрового предмета
    /// </summary>
    public string Category {  get; set; } = string.Empty;

    /// <summary>
    /// Описание игрового предмета в локализации на которой запущен сервер
    /// </summary>
    public string Description {  get; set; } = string.Empty;
}
